### PR TITLE
Cool New Cloaks!

### PR DESCRIPTION
**Pull request by RDS88-beeman**
## About The Pull Request

This adds two new cloak skins, a skin for the HoS and a skin for the QM. It also adds colorizers for these cloaks to the beecoin shop, at 20,000 coins each.

## Why It's Good For The Game

Drip good, beecoin sinks good

## Testing Photographs and Procedure
<details>
<summary>Screenshots</summary>
<details>
<summary>HOS Cloak</summary>

image
image
image

</details>

<details>
<summary>QM Cloak</summary>

image
image

</details>

</details>

## Changelog
:cl:
add: You can now obtain cavalier cloak colorizers from your local beecoin-based store! QM and HOS varieties available!
/:cl:

### CODE REVIEW

**Comment by ClownMoff** [comment]
No clown cloak :(

**Comment by RDS88-beeman** [comment]
another time (along with chaplain)

File: code/modules/client/loadout/loadout_colorizers.dm [hunk 1]
```diff
@@ -22,6 +22,16 @@
...
     allowed_roles = list(JOB_NAME_RESEARCHDIRECTOR)
 
+/datum/gear/colorizer/hoscloakknight
+    display_name = "Head of Security's Cloak Colorizer (Cavalier)"
+    path = /obj/item/colorizer/hoscloakknight
+    allowed_roles = list("Head of Security")
+
+/datum/gear/colorizer/qmcloakknight
+    display_name = "Quartermaster's Cloak Colorizer (Cavalier)"
+    path = /obj/item/colorizer/hoscloakknight
```
Comment (DrDuckedGoose) [inline code/modules/client/loadout/loadout_colorizers.dm]:
```suggestion
    path = /obj/item/colorizer/qmcloakknight
```

**Review by DrDuckedGoose — CHANGES_REQUESTED** [review]

**Comment by RDS88-beeman** [comment]
yeah good call i missed that

**Comment by PowerfulBacon** [comment]
Not a huge fan of the sprites, they seem very unclear

**Comment by PowerfulBacon** [comment]
Right now its too difficult to tell that the sprite is meant to represent a cape, and the colours on the HOS cape need work since it feels a bit coderspritey with the overly strong outline and grey body. They look too visually noisy

**Comment by EvilDragonfiend** [comment]
@PestoVerde322 I delegate my key to you if you think the style doesn't belong to ours.

**Comment by BlueHNT** [comment]
It looks nice, the colors are kind of missing its match for a lot of the HOS outfits. Still a lot better than the royal one by miles.

**Pull request closed without merging**